task-list/app/models/person.rb

```diff
@@ -0,0 +1,8 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def uncompleted
+    completed = self.tasks.where({:completed_at => nil})
```

Comment:
It might make sense to rename this variable since these are the pieces that are _not_ completed